### PR TITLE
add new task

### PR DESCRIPTION
**Pull request by MP91**
# Description

## Checklist

* [ ] Vehicle App can be started with dapr run and is connecting to vehicle data broker
* [ ] Vehicle App can process MQTT messages and call the seat service
* [ ] Vehicle App can be deployed to local K3D and is running
* [ ] Created/updated tests, if necessary. Code Coverage percentage on new code shall be >= 70%.
* [ ] Extended the documentation in Velocitas repo
* [ ] Extended the documentation in README.md

* [ ] Devcontainer can be opened successfully
* [ ] Devcontainer can be opened successfully behind a corporate proxy
* [ ] Devcontainer can be re-built successfully

* [ ] Release workflow is passing

### CODE REVIEW

File: .velocitas.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "packages": [
         {
             "name": "devenv-runtimes",
-            "version": "v2.2.5"
```
Comment (MP91) [inline .velocitas.json:5]: fix tag

Comment (kse3hi) [reply .velocitas.json:5]: Please fix before integration :)

Comment (MP91) [reply .velocitas.json:5]: that's why I added my comment :) now the correct tag is used

**Review by kse3hi — APPROVED** [review]
LGTM

**Pull request merged**